src/ui/FileMessageItemBody/__tests__/FileMessageItemBody.spec.js

```diff
@@ -1,14 +1,232 @@
 import React from 'react';
-import {shallow} from 'enzyme';
+import { mount } from 'enzyme';
 import renderer from 'react-test-renderer';
 
 import FileMessageItemBody from "../index";
 
+const createMockMessage = (process) => {
+  const mockMessage = {
+    type: 'image/png',
+    url: 'https://sendbird.com',
+    name: 'My name is name',
+    reactions: [],
+  };
+  return process ? process(mockMessage) : mockMessage;
+};
+
 describe('FileMessageItemBody', () => {
-  it('should do a snapshot test of the FileMessageItemBody DOM', function() {
-    const text = "example-text";
+  it('should have class names of own basic status', () => {
+    const className = 'classname-for-test';
+    const component = mount(
+      <FileMessageItemBody
+        className={className}
+        message={createMockMessage()}
+      />
+    );
+    expect(
+      component.find('.sendbird-file-message-item-body').hostNodes().exists()
+    ).toBe(true);
+    expect(
+      component.find(`.${className}.sendbird-file-message-item-body`).hostNodes().exists()
+    ).toBe(true);
+    expect(
+      component.find('.sendbird-file-message-item-body.outgoing').hostNodes().exists()
+    ).toBe(false);
+    expect(
+      component.find('.sendbird-file-message-item-body.incoming').hostNodes().exists()
+    ).toBe(true);
+    expect(
+      component.find('.sendbird-file-message-item-body.mouse-hover').hostNodes().exists()
+    ).toBe(false);
+    expect(
+      component.find('.sendbird-file-message-item-body.reactions').hostNodes().exists()
+    ).toBe(false);
+    expect(
+      component.find('.sendbird-file-message-item-body__file-icon').hostNodes().exists()
+    ).toBe(true);
+    expect(
+      component.find('.sendbird-file-message-item-body__file-icon__icon').hostNodes().exists()
+    ).toBe(true);
+    expect(
+      component.find('.sendbird-file-message-item-body__file-name').hostNodes().exists()
+    ).toBe(true);
+    expect(
```

Comment:
This part is easier to be a snapshot test, no?

Reply:
Its okay tho

Reply:
Actually, you are right. However, I wanted to show a comparison between this case and the other cases
so we will be able to know when some class names aren't displayed and not easily.